.github/workflows/github_release.yaml

```diff
@@ -0,0 +1,133 @@
+name: Release to GitHub
+
+on:
+  push:
+    branches:
+      - production
+
+jobs:
+  wf_vars:
+    name: "Set Workflow Variables"
+    runs-on: ubuntu-latest
+    outputs:
+      version: ${{ env.VERSION }}
+    steps:
+      - name: "Check out changes"
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 2
+      - name: Set Base Environment Variables
+        run: |
+          echo >> $GITHUB_ENV VERSION="v-$(date +%Y.%m.%d-%H%M%S)"
```

Comment:
Interesting! Is this an alternative way to set an output variable? Vs the `::set-output...` listed here: https://docs.github.com/en/actions/learn-github-actions/workflow-commands-for-github-actions#setting-an-output-parameter

Reply:
It's just the way to set an env variable, which is later set as output. This method is ideal if you want to leverage the env var in the current job. I'm no longer using the env var in the current job, so no actual benefit.

https://docs.github.com/en/actions/learn-github-actions/workflow-commands-for-github-actions#setting-an-environment-variable